src/chains/ethereum/ethereum/tests/api/eth/sendTransaction.test.ts

```diff
@@ -47,24 +47,58 @@ describe("api", () => {
       });
 
       describe("insufficient funds", () => {
-        it("returns an error when account has insufficient funds to send the transaction", async () => {
-          const p = await getProvider({
+        const types = ["0x0", "0x1", "0x2"] as const;
+        it("returns an error when account has insufficient funds to transfer the value", async () => {
+          const gasCost = 99967968750001;
```

Comment:
Does this really work? I'd be surprised if all three tx types have the same gas cost.

Reply:
it just has to be close enough. i've updated the var name to `approximateGasCost`